pages/about.html

```diff
@@ -4,9 +4,32 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="../styles/style.css" rel="stylesheet">
   <title>Document</title>
 </head>
 <body>
+    <header>
+      <nav id="navigation">
+        <a href="index.html" >Home</a>
+        <a href="portfolio.html">My portfolio</a>
+      </nav>
+      <h1>About Ja</h1>
+    </header>
+    <h2 id="projects">Introduction</h2>
+    <section class="about_container">
+      <p id="introduction">Hi! My name is Ja and I am software engineer in
+        training! Fun right? I know! My days are filled with studying but in my spare time, I love to play with
+        my kitty("Omi") and make jewelry for my small business, Pocketofjade! 
+        My absolute favorite things to do are cooking and exploring Las Vegas with my girlfriend.
+      </p>
+      <div id="cat"><img src="../images/kitty.jpg" alt="Omi"></div>
```

Comment:
We can do without the `div`s here. If you put the `id`s directly on the `img`s everything still works!

Reply:
Omi is totes adorbs! 😻